alpha/nodejs-chart/templates/buildconfig.yaml

```diff
@@ -19,8 +19,8 @@ spec:
     sourceStrategy:
       from:
         kind: ImageStreamTag
-        namespace: openshift
-        name: nodejs:12
+        namespace:
```

Comment:
These need to be defined via values files for both `namespace` and `name`

Reply:
Does the latest commit cover this?